### DISCUSSION TITLE
Lesson 4: Error Deploying Chainlink Data Feed Smart Contract on Sepolia Testnet Using WalletConnect in Remix

### DISCUSSION THREAD

**Discussion by komi786** [Cyfrin/foundry-full-course-cu#392] [Q&A]
I tried to deploy the contract on Sepolia testnet using "WalletConnect" environment option in Remix. I have 0.6 Eth in Sepolia testnet. The transaction fails with the following error. Can anyone help please

creation of DataConsumerV3 pending... creation of DataConsumerV3 errored: Missing or invalid parameters. Double check you have provided the correct parameters.

URL: https://rpc.sepolia.org/ Request body: {"method":"eth_sendTransaction","params":[{"from":"0x5b38da6a701c568545dcfcb03fcb875f56beddc4","data":"0x6.....","value":"0x0","gas":"0x2dc6c0","type":"0x2","maxPriorityFeePerGas":"0x9502F900","maxFeePerGas":"0x9898f120"}]}

Details: unknown account Version: viem@1.2.12

**Comment by robert-hoffmann** [comment]
Find a solution ? Same error here...

**Comment by maulobo** [comment]
Same error here...

**Comment by spomaria** [comment]
I'm having the same error too.

**Comment by Gloryjaw** [comment]
Please send your code with the parameter values.

**Comment by king-null** [comment]
you need to provide your code

**Comment by maulobo** [comment]
I fixed it using another RPC. I was using the real network, but you should use the testnet. Check your RPC_URL
